src/Microsoft.Health.Fhir.Core/Features/Operations/OperationsConstants.cs

```diff
@@ -18,5 +18,7 @@ public static class OperationsConstants
         public const string Reindex = "reindex";
 
         public const string ReindexContentTypeHeaderValue = "application/json";
+
+        public const string DataConvert = "dataConvert";
```

Comment:
If you look at examples in https://www.hl7.org/fhir/operationslist.html, then the compound name should be "data-convert". (Also, @ranvijaykumar, this applies to export)

Reply:
Thanks. Fixed